src/main/java/jenkins/plugins/openstack/compute/SlaveOptionsDescriptor.java

```diff
@@ -192,11 +192,6 @@ public FormValidation doCheckFloatingIpPool(
             @QueryParameter String value,
             @RelativePath("../../slaveOptions") @QueryParameter("floatingIpPool") String def
     ) {
-        if (Util.fixEmpty(value) == null) {
-            String d = getDefault(def, opts().getFloatingIpPool());
-            if (d != null) return FormValidation.ok(def(d));
-            return REQUIRED;
```

Comment:
Only this line needs to be changed to `return OK`. Removing it all causes default not showing up which is confusing.